v3/src/components/graph/components/binneddotplotdots.tsx

```diff
@@ -153,6 +156,13 @@ export const BinnedDotPlotDots = observer(function BinnedDotPlotDots(props: Plot
 
   usePlotResponders({pixiPoints, refreshPointPositions, refreshPointSelection})
 
+  // respond to binAlignment and binWidth changes because the axis may need to be updated
+  useEffect(function respondToGraphBinSettings() {
+    return mstAutorun(() => {
+      refreshPointPositions(false)
+    }, {name: "respondToGraphBinSettings"}, graphModel)
+  }, [dataset, graphModel, refreshPointPositions])
```

Comment:
This is likely to result in redundant calls to `refreshPointPositions()`. As an autorun, this function will trigger when any observable value referenced in `refreshPointPositions()` is changed. The `usePlotResponders()` call in line 257 sets up a bunch of other observers that could trigger at the same time. The reactions set up in `usePlotResponders()` are debounced so that (in theory) multiple calls won't result in multiple runs, but this call won't be debounced with the others. If the issue you're trying to solve here is really about `binAlignment` and `binWidth` settings, then perhaps a `reaction` that limited the response to those values would serve the purpose. I suspect there's a larger refactor lurking here in which there could be a single `autorun` in `use-plot.ts` which re-runs any time any observable value used in the function changes and then additional `reaction`s which just trigger on changes to properties not directly referenced in the `autorun`, but that's presumably outside the scope of this PR.